test/db.test.js

```diff
@@ -399,11 +399,13 @@ describe('Database', function () {
       process.removeAllListeners('uncaughtException');
 
       process.on('uncaughtException', function MINE (ex) {
+        process.removeAllListeners('uncaughtException');
```

Comment:
This was causing me a lot of grief while trying to debug actual test errors as they were all getting funneled into this `MINE` listener.